requirements.txt

```diff
@@ -61,6 +61,7 @@ hdfs==2.5.8               # via apache-beam
 html5lib==1.0.1           # via -r requirements.in
 httplib2==0.17.4          # via apache-beam, google-api-python-client, google-apitools, google-auth-httplib2, oauth2client
 idna==2.10                # via requests
+mailchimp3==3.0.14        # via -r requirements.in
```

Comment:
I didn't manually change this file. It was autogenerated when I changed `requirements.in`.